### PR TITLE
Dummy wheelchair node

### PR DESCRIPTION
**Pull request by Pandoro**
I quickly created a dummy wheelchair detection node as discussed with @lucasb-eyer . It would be great if @marc-hanheide and (others?) could have a quick look at it. 

Currently it doesn't really do a lot apart from publishing a random set of wheelchair detections. The main purpose for this is to discuss if we can agree upon the interfaces or if we need something completely different. The [README](https://github.com/Pandoro/strands_perception_people/tree/indigo-devel/wheelchair_detector) further explains some things.

### CODE REVIEW

**Comment by marc-hanheide** [comment]
good effort! I'll wait for the test to pass

File: wheelchair_detector/package.xml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<package>
+  <name>wheelchair_detector</name>
+  <version>0.0.0</version>
```
Comment (marc-hanheide) [inline wheelchair_detector/package.xml]: please make sure that this version string is precisely the same as in all other packages in this repo.

File: wheelchair_detector/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+#############
+## Install ##
+#############
+
+# all install targets should use catkin DESTINATION variables
+# See http://ros.org/doc/api/catkin/html/adv_user_guide/variables.html
+
+## Mark executable scripts (Python etc.) for installation
+## in contrast to setup.py, you can choose the destination
+# install(PROGRAMS
```
Comment (marc-hanheide) [inline wheelchair_detector/CMakeLists.txt]: please provide install targets for this package

**Comment by lucasb-eyer** [comment]
Thanks @Pandoro!

**Comment by Pandoro** [comment]
Thanks for the feedback :) Let's see if this check goes through :x.

**Comment by marc-hanheide** [comment]
great, thanks!

**Comment by cdondrup** [comment]
All this looks good. Just a few comments:
- Defaulting to scan is good but when integrated I would, like the leg detector, use the filtered laser scan which removes every laser measurement that corresponds to an obstacle in the static map. Might reduce false positives.
- no need for tracking as this will be added to the Bayes tracker. This, however, means that wheelchairs will be treated like other humans, which IMHO is still better that just obstacles.
- the orientation will be determined by the tracker based on the movement. Only works for moving wheelchairs and is not the real orientation of the wheelchair but the orientation of the velocity vector. Should be a close enough estimate though

**Comment by lucasb-eyer** [comment]
> - which removes every laser measurement that corresponds to an obstacle in the static map.

This may just as likely make results worse; it really depends on the model, we'll have to try and see.

**Comment by Pandoro** [comment]
Great thank you for the extra feedback. And I agree with Lucas on the background removal. I'm super happy too try this, but also a little skeptical about it.

**Comment by lucasb-eyer** [comment]
OK y'all, we are progressing really well and we have a nicely working detector (not in ROS yet) and are now writing things up for IROS.

In our experiments, we noticed the following:
- when the detection threshold is high, we get very few false positive while still detecting many wheelchairs.
- we can turn down that threshold A LOT, leading to the detection of almost all wheelchairs, but also many false-positive on walls, which would perfectly be avoided by static-map subtraction as suggested by @cdondrup. So now that we've seen how the net behaves, we agree with map-subtraction for the deployment :smile:

**Comment by lucasb-eyer** [comment]
ping @denisehe @TobKoer for the good news update above.

**Comment by marc-hanheide** [comment]
great news. If you've got something working, @Jailander, @gestom or @cdondrup might be able to borrow our colleague Kathrin's wheelchair for live tests ;-)

**Comment by lucasb-eyer** [comment]
I know they're just waiting for an excuse to play! Edit: and there's still quite some work for research code to production code.

**Comment by marc-hanheide** [comment]
:-)

**Pull request merged**